cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -122,8 +118,6 @@ spec:
         k8s-app: kube-dns
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        # TODO: Move this to the beta tolerations field below as soon as the Tolerations field exists in PodSpec
-        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}, {"key":"dedicated","value":"master","effect":"NoSchedule"}]'
```

Comment:
I intentionally removed it just for testing, as I noticed that on feb 8, all e2e tests were passing when  the file didn't have this toleration. So I am just verifying that. If this does not work, I will put it back.